### ISSUE TITLE
CQUI is missing from saved games

### ISSUE THREAD

**Issue by gungungunyyy** [Azurency/CQUI_Community-Edition#327]
I was running a Vanilla game with CQUI mod and it's no problem with in game But when I saved my current game and then I come back to play my saved game. CQUI is disappeared from additional content list and it seem like all my saved game got the same problem. Anyone got this problem? pls Help me out.

**Comment by rasputinoz** [comment]
Yes discovered same problem this morning. Saved games don't have CQUI in the list of installed mods

**Comment by Azurency** [comment]
Might be related to https://github.com/Azurency/CQUI_Community-Edition/commit/9d33d0d1c60c926b40e03137dd4ae72d421ae87a.
It's not listed because cqui is not changing the game itself but instead only the interface and it was decided some time ago to make it so that it does not affect save game. However if the mod is enabled it should be loading.

**Comment by rasputinoz** [comment]
IT not only not listed. it isn't activated in game any more. 

Works with New Games , but existing saved games not working

**Comment by fruttyx** [comment]
Same thing for me. Refilled saves do not work, CQUI is functional on a new game only.

**Comment by Azurency** [comment]
related to #348
